src/google-classroom-extractor/google_classroom_extractor/main.py

```diff
@@ -0,0 +1,222 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+import collections
+import logging
+import os
+import sys
+from datetime import datetime
+from typing import List
+
+import pandas as pd
+from dotenv import load_dotenv
+from googleapiclient.discovery import build, Resource
+from google.oauth2 import service_account
+from sqlalchemy import create_engine
+
+from google_classroom_extractor.api.courses import request_all_courses_as_df
+from google_classroom_extractor.api.coursework import request_all_coursework_as_df
+from google_classroom_extractor.api.students import request_all_students_as_df
+from google_classroom_extractor.api.submissions import request_all_submissions_as_df
+from google_classroom_extractor.api.usage import request_all_usage_as_df
+
+
```

Comment:
I think it would be nice to simplify a little bit the main.py script, or at least add some comments.

Reply:
Can refactor